packages/pluggableWidgets/range-slider-web/CHANGELOG.md

```diff
@@ -4,3 +4,7 @@ All notable changes to this widget will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+### Added
+- We added icons for dark mode.
+- We added dark mode to Structure mode preview.
```

Comment:
Is range-slider already released? If not, it's not really necessary to update changelog.

Reply:
Nope